tests/test_unittest/test_ir_maker.py

```diff
@@ -0,0 +1,14 @@
+import ast
+
+from gt4py.frontend.gtscript_frontend import IRMaker
+from gt4py.ir.nodes import BinaryOperator, BinOpExpr
+
+
+def test_AugAssign():
+    ir_maker = IRMaker(None, None, None, domain=None, extra_temp_decls=None)
+    aug_assign = ast.parse("a += 1").body[0]
+
+    _, result = ir_maker.visit_AugAssign(aug_assign)
+
+    assert isinstance(result.value, BinOpExpr)
+    assert result.value.op == BinaryOperator.ADD
```

Comment:
I think we should keep this test! It will pass once we merge with master after merging #198.

Reply:
I actually intended to contribute that one to #198, sorry for complicating things